purchase_request_to_procurement/views/purchase_request.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+<data>
```

Comment:
<code>data</code> tag is not needed if you put the <code>odoo</code> tag. The same for the other views.